From 1f778341bd600db36eeb7d8477e42f8f9bbc97f7 Mon Sep 17 00:00:00 2001
From: danie007 <danilselvan@gmail.com>
Date: Tue, 12 Jan 2021 15:59:20 +0530
Subject: [PATCH] Add redundant environment

Add #define CONFIG_SYS_REDUNDAND_ENVIRONMENT in imx8mm_evk.h to fix error
CONFIG_SYS_REDUNDAND_ENVIRONMENT is required for Mender to work

Signed-off-by: danie007 <danilselvan@gmail.com>
---
 include/configs/imx8mq_evk.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/configs/imx8mq_evk.h b/include/configs/imx8mq_evk.h
index 6ef8ec6083..fbe88264b9 100644
--- a/include/configs/imx8mq_evk.h
+++ b/include/configs/imx8mq_evk.h
@@ -10,6 +10,11 @@
 #include <asm/arch/imx-regs.h>
 #include "imx_env.h"
 
+/*
+* Added by Yocto as a fix to error CONFIG_SYS_REDUNDAND_ENVIRONMENT is required for Mender to work
+*/
+#define CONFIG_SYS_REDUNDAND_ENVIRONMENT
+
 #define CONFIG_NR_DRAM_BANKS 2
 
 #define CONFIG_SPL_MAX_SIZE		(148 * 1024)
